packages/graph/hash_graph/lib/graph/src/api/mod.rs

```diff
@@ -0,0 +1 @@
+pub mod web;
```

Comment:
What other types of API could you envision? I'm a little unsure about the naming of `web`. If we're generation OpenAPI Spec I'm assuming we are a true REST server so maybe we should just call this `mod rest`? 

Some of the other API approaches will probably still go over http and such, which is why I'm unsure about this

Reply:
I could rename to rest instead, yeah. The idea was that something like `gRPC` could be here as well.

Reply:
Addressed in https://github.com/hashintel/hash/pull/750/commits/a462cf41a1c8480f6138ac2eed9cd00cde4ccc4b